src/main/java/org/sagebionetworks/bridge/notification/worker/WorkerConfig.java

```diff
@@ -82,48 +83,45 @@ public void setExcludedDataGroupSet(Set<String> excludedDataGroupSet) {
     }
 
     /**
-     * Messages to send if the participant misses a cumulative total of activities during this study burst, keyed by
-     * data group. The keys in this set should match the keys in {@link #getRequiredDataGroupsOneOfSet}.
+     * Messages to send if the participant misses a cumulative total of activities during this study burst. Worker
+     * will pick one at random.
      */
-    public Map<String, String> getMissedCumulativeActivitiesMessagesByDataGroup() {
-        return missedCumulativeActivitiesMessagesByDataGroup;
+    public List<String> getMissedCumulativeActivitiesMessagesList() {
+        return missedCumulativeActivitiesMessagesList;
     }
 
-    /** @see #getMissedCumulativeActivitiesMessagesByDataGroup */
-    public void setMissedCumulativeActivitiesMessagesByDataGroup(
-            Map<String, String> missedCumulativeActivitiesMessagesByDataGroup) {
-        this.missedCumulativeActivitiesMessagesByDataGroup = missedCumulativeActivitiesMessagesByDataGroup != null ?
-                ImmutableMap.copyOf(missedCumulativeActivitiesMessagesByDataGroup) : ImmutableMap.of();
+    /** @see #getMissedCumulativeActivitiesMessagesList */
+    public void setMissedCumulativeActivitiesMessagesList(List<String> missedCumulativeActivitiesMessagesList) {
+        this.missedCumulativeActivitiesMessagesList = missedCumulativeActivitiesMessagesList != null ?
```

Comment:
As this isn't initialized to null, would this be clearer/shorter:
if (missedCumulativeActivitiesMessagesList != null) {
    this. missedCumulativeActivitiesMessagesList = missedCumulativeActivitiesMessagesList;
}

Reply:
As per IM conversation, the semantics of this setter are that (a) null values are replaced with empty lists and (b) we copy the list so that changes in the original list don't get reflected through. I think this is the cleanest way to do this.